### PR TITLE
CHANGELOG: release 0.31.0

### PR DESCRIPTION
**Pull request by smallhive**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,22 +4,29 @@ This document outlines major changes between releases.
 
 ## [Unreleased]
 
+### Added
+
 ### Changed
-- Single `rpc_endpoint` config option replaced with multiple endpoints `fschain.endpoints` option (#968)
 
-### Updating from 0.30.1
```
Comment (roman-khimov) [inline CHANGELOG.md:10]: Please mention that it's only compatible with 0.43.0+ node. And old token boxes are incompatible with this version.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,22 +4,29 @@ This document outlines major changes between releases.
...
+### Updated
+
+## [0.31.0] - 2024-08-14
+
+### Added
+- Use HKDF salt and info (#972)
+- Allow addresses to be used in EACLs (#976)
+
+### Updated
+- github.com/nspcc-dev/neofs-sdk-go dependency to the latest version
```
Comment (roman-khimov) [inline CHANGELOG.md:22]: Reference?

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,22 +4,29 @@ This document outlines major changes between releases.
...
-```
-to the new one
-```yaml
-fschain:
-  endpoints:
-    - http://some-host:30333
-```
+### Updated
+
+## [0.31.0] - 2024-08-14
```
Comment (roman-khimov) [inline CHANGELOG.md:15]: Date will need to be fixed.

File: CHANGELOG.md [hunk 4]
```diff
@@ -583,4 +590,5 @@ releases.
 [0.29.0]: https://github.com/nspcc-dev/neofs-s3-gw/compare/v0.28.2...v0.29.0
 [0.30.0]: https://github.com/nspcc-dev/neofs-s3-gw/compare/v0.29.0...v0.30.0
 [0.30.1]: https://github.com/nspcc-dev/neofs-s3-gw/compare/v0.30.0...v0.30.1
-[Unreleased]: https://github.com/nspcc-dev/neofs-s3-gw/compare/v0.30.1...master
+[0.31.0]: https://github.com/nspcc-dev/neofs-s3-gw/compare/v0.30.0...v0.31.0
```
Comment (roman-khimov) [inline CHANGELOG.md:593]: v0.30.1...v...

File: CHANGELOG.md (hunk 1 above)
Comment (smallhive) [reply CHANGELOG.md:10]: Added

File: CHANGELOG.md (hunk 2 above)
Comment (smallhive) [reply CHANGELOG.md]: Updated

File: CHANGELOG.md (hunk 3 above)
Comment (smallhive) [reply CHANGELOG.md]: Changed

File: CHANGELOG.md (hunk 4 above)
Comment (smallhive) [reply CHANGELOG.md]: Updated

**Comment by roman-khimov** [comment]
Please update, we're rolling it out today.

File: CHANGELOG.md [hunk 5]
```diff
@@ -23,19 +41,13 @@ This document outlines major changes between releases.
 - google.golang.org/grpc dependency from v1.62.0 to v1.62.1 (#811)
 - github.com/minio/sio dependency from v0.3.0 to v0.4.0 (#811)
 
-### Updating from 0.30.1
+### Fixed
+- Save hashless object header in SpitV2 (#974)
+- Clean actual object version (#970)
 
-Notice that the configuration parameter `fschain.endpoints` contains multiple endpoints.
```
Comment (roman-khimov) [inline CHANGELOG.md:28]: Why is this removed?

Comment (smallhive) [reply CHANGELOG.md:28]: That’s an excellent question. It was rebased without conflict. Reverted

**Review by roman-khimov — APPROVED** [review]
